doc/ref_model/chapters/chapter03.md

```diff
@@ -214,10 +214,14 @@ The hardware infrastructure manager needs to support the following functional as
 <a name="3.4"></a>
 ## 3.4 Hardware Infrastructure Resources
```

Comment:
```suggestion
## 3.4 Cloud Infrastructure Hardware Resources
```

Reply:
Chapter 3.1 points out the layering between the Virtualization Infrastructure Layers and the Hardware Virtualization Infrastructure Layer that both are part of the Cloud Infrastructure.
If we start making the very deep explanations of pre-pending everything with Cloud Infrastructure we will have to change in many places and it will not be very much clearer.

Reply:
My comment is just about using consistent wording, not adding more explanation.
This chapter is not describing the Hardware Infrastructure Resources.